src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ConfigurationController.cs

```diff
@@ -19,7 +19,7 @@ public ConfigurationController(IConfiguration configuration)
         [HttpGet("api/config")]
         public ActionResult<Dictionary<string, string>> GetConfig()
         {
-            return Ok(BuildConfigurationValues());
+            return new ActionResult<Dictionary<string, string>>(BuildConfigurationValues());
```

Comment:
I think you can just `return BuildConfigurationValues();` because of the `ActionResult<T>` implicit operator that's built-in and will do it for you.

Reply:
Hey thanks @benoutram - apparently not, as it thinks you're attempting to use the protected (non-generic) ActionResult constructor.